cppcache/src/PdxType.cpp

```diff
@@ -53,9 +54,13 @@ PdxType::PdxType(PdxTypeRegistry& pdxTypeRegistry,
       m_isVarLenFieldAdded(false),
       m_remoteToLocalFieldMap(nullptr),
       m_localToRemoteFieldMap(nullptr),
-      m_noJavaClass(false),
+      m_noJavaClass(!expectDomainClass),
```

Comment:
Let's invert the member variable to be positive and reflect the usage. So if it is the "expect domain class" member now let's rename it so.

Reply:
Good point :)